### DISCUSSION TITLE
display tab when switching tab with key binds in compact mode

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#761] [Ideas]
### What feature would you like to see?

in compact mode,
It would be nice if can display the tab list when switching tab with keybind like ctrl-tab and ctrl-shift-tab. It can give the user(ei. me) a visual cue to what am I switching to.

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/691_

**Comment by ijontan** [comment]
Thanks for the feature, I just updated, really glad seeing it being implemented!

**Comment by DatHypnoboi** [comment]
It would also be nice to have this happen when opening a link in a new tab, my middle click is intermittent so I never really know if it worked until I mouse over the sidebar

**Comment by obrassard** [comment]
I also think this would be a nice addition to Zen !
